src/Microsoft.DotNet.XHarness.CLI/Commands/Android/AndroidGetDeviceCommand.cs

```diff
@@ -42,8 +42,6 @@ protected override Task<ExitCode> InvokeInternal(ILogger logger)
                 return Task.FromResult(ExitCode.PACKAGE_NOT_FOUND);
             }
 
-            _arguments.Verbosity = LogLevel.Critical;
```

Comment:
I think it might be better to initialize the arguments with the high level and leave the possibility of the user overriding it but keep it so that when using the command without it, it makes sense (so UDID printed only).

Also why not `Error` level? Is there a reason for `Critical`?

Reply:
Ah, I didn't see your previous comment. Let me have a look.

Reply:
You should be able to do following above:
```
        private readonly AndroidGetDeviceCommandArguments _arguments = new()
        {
            Verbosity = LogLevel.Error
        };
```

Reply:
ahh, got it! will change to `Error` too